src/index.html

```diff
@@ -39,18 +39,14 @@ <h1 class="hero__title"> We make your music sound extraordinary.</h1>
 	<!-- Section of the hero images -->
 
 	<!-- The black background has me questioning if the blue art inside should be in it or hero-imgs should just be set to absolute. -->
-	<main class="hero-imgs">
+	<main class="main">
 		<div class="container">
-			<div class="blue-art">
-				<img src="./assets/bg-pattern-2.svg" alt="">
-			</div>
-
-			<div class="app-img">
-				<img src="./assets/illustration-app.png" alt="">
+			<div class="main__illustration">
+				<img src="./assets/illustration-app.png" alt="Screenshot of the equalizer application showing levels of an equaliser, a decible volume, a selection of tone and an apply settings button">
```

Comment:
So you want to change this so that the black background has a section on it own?

Reply:
Sorry you mean the background pattern?

Reply:
The background pattern should be on the whole main section, if you check the design you see it's always top center and behind the content.